Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBiasHI_cff.py

```diff
@@ -6,56 +6,132 @@
 
 import EventFilter.HcalRawToDigi.HcalRawToDigi_cfi
 hcalDigiAlCaMB = EventFilter.HcalRawToDigi.HcalRawToDigi_cfi.hcalDigis.clone()
-import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hbhe_cfi
-hbherecoNoise = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hbhe_cfi.hbheprereco.clone()
-import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hf_cfi
-hfrecoNoise = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hf_cfi.hfreco.clone()
-import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hf_cfi
-hfrecoMBspecial = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hf_cfi.hfreco.clone()
-import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_ho_cfi
-horecoNoise = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_ho_cfi.horeco.clone()
+
+hcalDigiAlCaMB.firstSample = 0
+hcalDigiAlCaMB.lastSample = 9
+hcalDigiAlCaMB.InputLabel = 'rawDataCollector'
+
 #add GT digi:
 import EventFilter.L1GlobalTriggerRawToDigi.l1GtUnpack_cfi
 gtDigisAlCaMB = EventFilter.L1GlobalTriggerRawToDigi.l1GtUnpack_cfi.l1GtUnpack.clone()
 
+gtDigisAlCaMB.DaqGtInputTag = 'source'
+
 import HLTrigger.HLTfilters.hltHighLevel_cfi
 hcalminbiasHLT =  HLTrigger.HLTfilters.hltHighLevel_cfi.hltHighLevel.clone(
 #    HLTPaths = ['HLT_HcalPhiSym'],
     eventSetupPathsKey='HcalCalMinBiasHI',
     throw = False #dont throw except on unknown path name 
 )
 
-seqALCARECOHcalCalMinBias = cms.Sequence(hcalminbiasHLT*hcalDigiAlCaMB*gtDigisAlCaMB*hbherecoNoise*hfrecoNoise*hfrecoMBspecial*horecoNoise)
+import RecoLocalCalo.HcalRecProducers.HBHEPhase1Reconstructor_cfi
+hbherecoNoise = RecoLocalCalo.HcalRecProducers.HBHEPhase1Reconstructor_cfi.hbheprereco.clone(
```

Comment:
Hi @bsunanda - can the duplicate module definitions (eg, hbherecoNoise) be consolidated into a cfi that both cffs can use?